src/components/MerchantsPage/gam/styles.module.scss

```diff
@@ -176,3 +183,115 @@
     font-size: 16px;
   }
 }
+
+.noCampaign {
+  align-items: center;
+  border: 1px solid #e5e5e5;
+  display: grid;
+  grid-template-columns: 2fr 1fr;
+  padding: 30px;
+
+  @media (max-width: 900px) {
+    grid-template-columns: 1fr;
+    margin-bottom: 15px;
+
+    & > div {
+      margin-bottom: 20px;
+    }
+  }
+}
+
+.noCampaignText {
+  :first-child {
+    font-style: normal;
+    font-weight: 600;
+    font-size: 24px;
+    line-height: 33px;
+    letter-spacing: 0.02em;
+
+    @include for-phone-only {
+      font-size: 14px;
+      line-height: 19px;
+    }
+  }
+
+  :last-child {
+    font-size: 15px;
+
+    @include for-phone-only {
+      font-size: 12px;
+    }
+  }
+}
+
+.giftButton {
+  justify-self: flex-end;
+  text-align: center;
+  width: 240px;
+  letter-spacing: 0.15em;
+  cursor: pointer;
+  margin-bottom: 18px;
+  font-family: 'Open Sans', 'Helvetica Neue', sans-serif;
+  font-weight: 700;
+  font-size: 16px;
+  letter-spacing: 0.05em;
+  width: 100%;
+
+  @media (max-width: 550px) {
+    font-size: 14px;
+    width: 100%;
+  }
+}
+
+// instructions modal styling
+
+.modalSub {
+  background-color: $standard-red;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 20px 10px;
+  margin: 20px 0;
+}
+
+.modalInstructions {
+  width: 100%;
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  grid-column-gap: 5px;
+
+  @media (max-width: 900px) {
+    grid-auto-flow: column;
+    grid-template-columns: repeat(2, 1fr);
+    grid-template-rows: repeat(2, 1fr);
+    grid-row-gap: 15px;
+    justify-items: center;
+  }
+}
+
+.modalStep {
+  display: grid;
+  grid-template-columns: min-content 1fr;
+  grid-column-gap: 5px;
+  width: 145px;
+
+  img {
+    width: 35px;
+  }
+}
+
+.modalStepText {
+  p {
+    color: white;
+    margin: 0;
+    font-size: 11px;
+  }
+
+  :first-child {
+    text-transform: uppercase;
+  }
+
+  :last-child {
+    font-weight: 600;
+  }
```

Comment:
These first and last child things are starting to be a bit unclear to me since it's used a lot here. Would it be better to just define an explicit class for the first and last class of these steps?